packages/rainbowkit/src/wallets/walletConnectors/xdefiWallet/xdefiWallet.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { InjectedConnector } from 'wagmi/connectors/injected';
+import { Chain } from '../../../components/RainbowKitProvider/RainbowKitChainContext';
+import { Wallet } from '../../Wallet';
+
+declare global {
+  interface Window {
+    xfi: any;
+  }
+}
+
+export interface XDEFIWalletOptions {
+  chains: Chain[];
+  shimDisconnect?: boolean;
+}
+
+export const xdefiWallet = ({
+  chains,
+  shimDisconnect,
```

Comment:
Should pass through `InjectedConnectorOptions`. Reference other connector implementations.